packages/jest-circus/src/utils.js

```diff
@@ -240,7 +241,12 @@ export const callAsyncCircusFn = (
 
 export const getTestDuration = (test: TestEntry): ?number => {
   const {startedAt} = test;
-  return startedAt ? Date.now() - startedAt : null;
```

Comment:
Previously it was missing completely when time was fakes as `startedAt === 0` which was falsy. using `hrtime` it changed to always say `0ms`. The babel plugin ensures we reference the real `process.hrtime` now

Reply:
Isn't the millisecond resolution from `Date.now` enough or is there any other reason to use `process.hrtime`?

Reply:
In theory it's more accurate, although I don't think that matters in practice.
Any reason to prefer `Date`?

Reply:
If we don't need a more accurate precision than milliseconds we can stick to Date. I'd prefer it for simplicity.

Reply:
Changed back, but fixed the `startedAt === 0` case